bin/test_coverage_advanced.dart

```diff
@@ -48,12 +56,18 @@ Future main(List<String> arguments) async {
   generateMainScript(packageRoot, testFiles);
   print('Generated test-all script in test/.test_coverage.dart. '
       'Please make sure it is added to .gitignore.');
-  await runTestsAndCollect(Directory.current.path, port).then((_) {
+  await runTestsAndCollect(Directory.current.path, port, printOutput: options.wasParsed("print-test-output")).then((_) {
     print('Coverage report saved to "coverage/lcov.info".');
   });
   final lcov = File(path.join(packageRoot.path, 'coverage', 'lcov.info'));
   final lineCoverage = calculateLineCoverage(lcov);
   generateBadge(packageRoot, lineCoverage);
   final coveragePct = (lineCoverage * 100).floor();
   print('Overall line coverage rate: $coveragePct%.');
+  final minCoverage = int.parse(options["min-coverage"]);
+  if (coveragePct < minCoverage) {
+    print('Overall coverage $coveragePct is small then minimal coverage $minCoverage');
```

Comment:
This should read `Overall coverage $coveragePct is less than minimum required coverage $minCoverage`

Reply:
fixed